### ISSUE TITLE
Analytics Implementation or QA Support for [Authenticated Experience My VA - click-tracking on Nametag]

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#66434]
# Google Analytics Implementation or QA Request

## What this form is for

<details>
<summary>More Information</summary>
<div>
    <br>
<p>Use this template to request Google Analytics implementation or QA for your product. Please fill out all brackets.</p>
    
> _Please see the following information for implementation timeline:_
> New Teams: Implementation will usually be completed in 4 sprints
> 
> - Sprint 1: Discovery/Orientation
> - Sprint 2: FE Implementation
> - Sprint 3: Tagging Implementation
> - Sprint 4: QA
> 
> All other teams: Implementation will usually be completed in 2 sprints
> 
> - Sprint 1: Discovery and FE Implementation
> - Sprint 2: Tagging Implementation and QA

</div>
</details>

### Required
- **Is this request for **Google Analytics implementation** or **QA**?**

     - [X] Google Analytics implementation
     - [ ] QA

- **Who is this request for?**

    - Authenticated Experience My VA
    
    
- **Provide the link(s) for Google Analytics implementation or QA**
  
    - [Insert link(s)]
    
- **When is the launch date?**
    - As soon as possible - this has been live in production for some time without tracking
   
**What interactions need to be tracked?** _NOTE: [Design System Components](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Design-System-Components.1840808040.html) are automatically tracked_

- Nametag banner is an existing feature that lives at the top of My VA and Profile within the authenticated experience. We don't have any kind of click-tracking on it and need to add it the link that navigates the user to "view your VA disability ratings" page (`va.gov/disability/view-disability-rating/rating`)
- When we're able to reach the disability rating API we show the user's disability rating, "n% service connected" and that text is a link to the page referenced above.
- If the disability rating API call fails, we just show a link to that same page but the text reads "view disability rating"

**Funnel:** 
1. Log into staging with user 36
2. Once on My VA (redirect should land you there post-login) see the light blue nametag "banner" at the top of the page just below the main va.gov header. 
3. You'll see the users' name, branch of service in text and branch's seal, followed by the disability rating. 
4. Users see the same nametag banner on profile as well (va.gov/profile, and all subpages of profile)     

User 36:
![Screenshot 2023-09-27 at 11 46 12 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/45603961/826ab456-06d6-48ec-bea0-88b1bb1bc47c)

(**Note:** Depending on the test user they may not have a branch and/or disability rating. If a veteran or test user has no service disability we don't show anything in the banner about disability. For example see test user 10)

    - [Insert other interactions - please review the [Design System Components](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Design-System-Components.1840808040.html) and [GA Data Dictionary](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Google-Analytics-Data-Dictionary.1810464912.html)  to make sure these interactions aren't already tracked]

### Optional
- **If applicable, what test users are needed?**  _NOTE: Do not put staging credentials in this ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository_

    - [Insert test user]

- **Any additional comments**

    - [Insert comments]

> Please leave the following blank

## Acceptance criteria
- [ ] All appropriate issue tagging is completed
- [ ] QA has been completed
- [ ] Events have been added to `Product` and `Benefit Hub` Content Groups

## Additional information

Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Analytics Request Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Analytics-request.1782120453.html) on Platform website for more information about the Collaboration Cycle.

**Comment by ajakabcin** [comment]
hey @michelle-dooley! I'll take a look at this and discuss with Samara. Jamie helped us with a similar report recently as well. We're also confused though because we're using standard link text in the nametag (it's not new) which I thought was a component and therefore should have tracking built in I thought, but maybe I'm misunderstanding how that works?

**Comment by ajakabcin** [comment]
@michelle-dooley we can close this or put it on hold for now and use the previous page path. Eventually we'd like tracking on that though when possible. Thanks!
